docker/Makefile

```diff
@@ -3,18 +3,21 @@ ROOT_IMAGE ?= alpine:3.13
 CERT_IMAGE := alpine:3.13
 GOLANG_IMAGE := golang:1.15-alpine
 
-BASE_IMAGE := localhost/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
-DEBUG_IMAGE := localhost/debugimg:$(VERSION)-$(shell echo $(GOLANG_IMAGE) | tr : -)
+BASE_IMAGE := localhost:5000/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
+DEBUG_IMAGE := localhost:5000/debugimg:$(VERSION)-$(shell echo $(GOLANG_IMAGE) | tr : -)
```

Comment:
Yes, you're right, I'll change the names of base image and debug image to stable local names. Thanks!

Reply:
Hi @yurishkuro , I've changed the names of `baseimg` and `debugimg` to stable local names. I still set the `debug_image` arg to `golang:1.15-alpine` for building multi-arch images in `build-all-in-one-image.sh` and `build-upload-docker-images.sh`, since the built `debugimg` doe not support `s390x` yet (due to lack of `go-delve` support on s390x). For building amd64 images, the `debug_image` arg is set to the stable local name.

Reply:
Hi @yurishkuro , I've made some changes to remove cross-script dependencies of `debug_image` image names. Please take a look when you are available. Thanks!